backend/build.sbt

```diff
@@ -5,12 +5,23 @@ version := "0.1"
 scalaVersion := "2.13.6"
 
 val Http4sVersion = "1.0.0-M21"
-val CirceVersion = "0.14.0-M5"
+val CirceVersion = "0.14.3"
+val Log4JVersion = "2.0.1"
 libraryDependencies ++= Seq(
+  "org.slf4j"       %  "slf4j-api"           % Log4JVersion,
+  "org.slf4j"       %  "slf4j-nop"           % Log4JVersion,
```

Comment:
Since no logger warning is now displayed due to those dependencies we should try to add some kind of information that service has started.

Reply:
Updated.